uint8 LEFT = 0
uint8 RIGHT = 1

string command
int32 camera
---
